engine/src/main/java/io/camunda/zeebe/engine/processing/resource/ResourceDeletionProcessor.java

```diff
@@ -56,8 +56,9 @@ public void processNewCommand(final TypedRecord<ResourceDeletionRecord> command)
     tryDeleteResources(command);
 
     final long eventKey = keyGenerator.nextKey();
+    stateWriter.appendFollowUpEvent(eventKey, ResourceDeletionIntent.DELETING, value);
+    responseWriter.writeEventOnCommand(eventKey, ResourceDeletionIntent.DELETING, value, command);
     stateWriter.appendFollowUpEvent(eventKey, ResourceDeletionIntent.DELETED, value);
```

Comment:
:thought_balloon: I assume this will get refactored when we support deleting BPMNs. Looking at this processor it looks like we still don't, so :shrug:

Reply:
That's right, BPMN will be added later.